fava/static/javascript/codemirror-mode-beancount.js

```diff
@@ -38,7 +38,7 @@ CodeMirror.defineSimpleMode('beancount', {
       token: 'whitespace',
     },
     {
-      regex: /[*!&#?%PSTCURM]|txn/,
+      regex: /[*!&?%PSTCURM]|txn/,
```

Comment:
I'd prefer 2). The regex above in this line is copied from beancount's `lexer.l` so changing it risks reintroducing this bug if one tries to match `lexer.l` in the future.